src/App.jsx

```diff
@@ -14,33 +16,80 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+export const App = () => {
+  const [sortBy, setSortBy] = useState(null);
+  const [reverseSort, setReverseSort] = useState(false);
+  const [goods, setGoods] = useState([...goodsFromServer]);
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+  return (
+    <div className="section content">
+      <div className="buttons">
+        <button
+          onClick={() => {
+            setSortBy('alphabetically');
+            setGoods(goods.sort((a, b) => a.localeCompare(b)));
+          }}
+          type="button"
+          className={classNames('button is-info', {
+            'is-light': sortBy !== 'alphabetically',
+          })}
+        >
+          Sort alphabetically
+        </button>
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+        <button
+          onClick={() => {
+            setSortBy('length');
+            setGoods(goods.sort((a, b) => a.length - b.length));
+          }}
```

Comment:
Move to separate function  (transfer the function to a variable, and here only write the name of the function)
Fix all cases if present